packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -95,7 +95,13 @@ export class PeoplePage {
 			elementHandle
 		);
 
-		await this.page.check( selectors.deletedUserContentAction( 'delete' ) );
+		// Native `page.check` sometimes fails here. Instead, click on the radio and wait for the
+		// Delete user button to become enabled.
+		await this.page.click( selectors.deletedUserContentAction( 'delete' ) );
+		await this.page.waitForSelector( `${ selectors.deleteUserButton }[disabled=""]`, {
+			state: 'detached',
+		} );
```

Comment:
I don't think we should need this line right? The click action below will wait until the `deleteUserButton` is enabled by default.

I had kind of been imagining a `waitForSelector` for the checked state of the radio button, which would wait until the state is updated before proceeed.

You should be able to do that with the `:checked` psuedo-selector...
```typescript
await this.page.waitForSelector( `${ selectors.deletedUserContentAction( 'delete' ) }:checked` );
```

Reply:
My intention was to wait until the disabled marker for the `deleteUserButton` was removed, which would signify that a valid selection was made in the radio.

I suppose something similar can be done by either just waiting for the button to be enabled, or using that pseudo-CSS selector. I can try either or.

Reply:
Pushed changes in 601ede455a16dfa891c3c05b0a11d857ae037edd